llvm/test/CodeGen/AMDGPU/flat_atomics_i64_noprivate.ll

```diff
@@ -0,0 +1,6911 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=amdgcn -mcpu=bonaire < %s | FileCheck -check-prefix=GCN1 %s
```

Comment:
Why GCN1 and GCN2? GFX7 and GFX8 are easier to understand.

Reply:
Copied from the other test